huggingface/pytorch/inference/docker/1.9/py3/sdk1.16.1/Dockerfile.neuron

```diff
@@ -0,0 +1,138 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG MMS_VERSION=1.1.4
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.8.10
```

Comment:
Do we need python version 3.8 for HF. The recommended/tested python versions with torch-neuron are 3.6 & 3.7. If there is no hard requirement for 3.8 then can we use 3.7 like in [neuron](https://github.com/aws/deep-learning-containers/blob/master/pytorch/inference/docker/1.9/py3/sdk1.16.1/Dockerfile.neuron)

Reply:
Changed to Python3.7. 
I wouldn't go with Python3.6 since it is deprecated End of Month